### PR TITLE
adding a private "read it later" library to half of the new users who…

### PR DESCRIPTION
**Pull request by eishay**
… would join, adding an experiment for tracking.

### CODE REVIEW

**Comment by eishay** [comment]
@andrewconner @leogrim

File: kifi-backend/modules/common/app/com/keepit/model/Library.scala [hunk 1]
```diff
@@ -302,6 +302,7 @@ object LibraryKind {
   case object SYSTEM_MAIN extends LibraryKind("system_main", 0)
   case object SYSTEM_SECRET extends LibraryKind("system_secret", 1)
   case object SYSTEM_PERSONA extends LibraryKind("system_persona", 2)
+  case object SYSTEM_READ_IT_LATER extends LibraryKind("system_read_id_later", 1)
```
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/model/Library.scala]: It'll come before user created libraries?

Comment (eishay) [inline kifi-backend/modules/common/app/com/keepit/model/Library.scala]: Yes, what do you think?

Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/model/Library.scala]: Probably just let it be mixed in, like persona libraries. They'll see it before it gets buried, and doesn't get annoying.

**Comment by eishay** [comment]
ok

**Pull request merged**